git_machete/github.py

```diff
@@ -216,13 +216,15 @@ def __fire_github_api_request(method: str, path: str, token: Optional[str], requ
                 'Visit `https://github.com/settings/tokens` to generate a new one.')  # TODO (#164): make dedicated exception here
         elif err.code == http.HTTPStatus.TEMPORARY_REDIRECT:
             current_repo_and_org = get_repo_and_org_names_by_id(err.headers['Location'].split('/')[4])
-            new_path = f'/repos/{current_repo_and_org}/{"/".join(path.split("/")[4:])}'
+            pulls_or_issues_api_suffix = "/".join(path.split("/")[4:])
```

Comment:
+1, much clearer now, I wouldn't guess that it might be either `pulls` or `issues`...

Reply:
Actually, won't it always be `pulls`? 🤔 why can it end up being `issues`? 😯

Reply:
Awww okay, coz this method is generic right? can be called not just for creating a new PR but for other cases as well?

Reply:
Yes, exactly, it can be used for other cases as well